tests/acceptance/docker/Makefile

```
@@ -42,8 +42,8 @@ help:
 	@echo -e "oCIS feature tests and test suites can be found here:"
 	@echo -e "\thttps://github.com/owncloud/ocis/tree/master/tests/acceptance/features"
 	@echo
-	@echo -e "ownCloud feature tests and test suites can be found here:"
-	@echo -e "\thttps://github.com/owncloud/core/tree/master/tests/acceptance/features"
+	@echo -e "ownCloud feature tests and test suites that are applicable for oCIS can be found here and they start with prefix coreApi-:"
```

what can be the appropriate way to say this?

---

```suggestion
	@echo -e "test suites that test ownCloud 10 API core compatibility are found here and they start with prefix coreApi-:"
```

Maybe something like this?